text_collector_examples/apcaccess.sh

```diff
@@ -160,10 +164,10 @@ echo -n 'node_apc_ups_info{'
 write_kv 'model' "$ups_model"
 write_kv 'name' "$ups_name"
 write_kv 'status' "$ups_status"
-write_kv 'battery_date' "$ups_battery_date"
+write_kv 'battery_date_seconds' "$ups_battery_date"
 write_kv 'last_transfer_reason' "$ups_last_transfer_reason"
-write_kv 'shutdown_time_min' "$ups_shutdown_time_min"
-write_kv 'shutdown_time_max' "$ups_shutdown_time_max"
-write_kv 'shutdown_charge_min' "$ups_shutdown_charge_min"
+write_kv 'shutdown_time_min_seconds' "$ups_shutdown_time_min"
+write_kv 'shutdown_time_max_seconds' "$ups_shutdown_time_max"
+write_kv 'shutdown_charge_min_voltage' "$ups_shutdown_charge_min"
```

Comment:
I think this should be `_volts`.  See: [Voltage vs Volts](https://electronics.stackexchange.com/a/182430).